scripts/gas-refund-program/staking/2.0/ERC20StateTracker.ts

```diff
@@ -101,24 +102,25 @@ export default class ERC20StateTracker extends AbstractStateTracker {
       `loadStateChanges: loading psp balance related events for all pools`,
     );
 
-    const events = (this.transferEvents = (await this.contract.queryFilter(
+    this.transferEvents = await queryFilterBatched(
```

Comment:
any valid reason why it's a class attribute now ?

Reply:
I did this change because I did not understand what we do:
`const events = (this.transferEvents = `

We did not do any sort of unwanted operations on the class member so I supposed that this is some remaining code from a previous implementation.

Reply:
oh you're right I just noticed, I think see any valid reason, on me